draft-ietf-quic-transport.md

```diff
@@ -2077,9 +2084,13 @@ genuine migrations.  Changing port number can cause a peer to reset its
 congestion state (see {{migration-cc}}), so the port SHOULD only be changed
 infrequently.
 
-An endpoint that exhausts available connection IDs cannot migrate.  To ensure
-that migration is possible and packets sent on different paths cannot be
-correlated, endpoints SHOULD provide new connection IDs before peers migrate.
+An endpoint that exhausts available connection IDs cannot migrate.  Similarly,
+an endpoint is unable to respond to probes or an attempt by its peer to migrate
```

Comment:
This sentence parses oddly for me, I assume it's in the context of the previous sentence, but maybe something like
```
An endpoint that exhausts available connection IDs cannot probe new paths or initiate 
migration, nor can it respond to probes or attempts by its peer to migrate.
```
and replace both sentences.
(Not providing that as a suggested change since it'll mess up the wrapping, sorry!)